rhasspysupervisor/__init__.py

```diff
@@ -1905,7 +1905,7 @@ def get_speakers(
     if sound_system == "aplay":
         play_command = ["aplay", "-q", "-t", "wav"]
         list_command = ["aplay", "-L"]
-        sound_device = profile.get("sounds.arecord.device", "").strip()
+        sound_device = profile.get("sounds.aplay.device", "").strip()
```

Comment:
this technically breaks backwards compat, could check for both and defer to arecord config if it exists, but :woman_shrugging: don't think it's worthwhile to here